sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/healthcare/FhirIOTestUtil.java

```diff
@@ -136,7 +136,7 @@ public static void tearDownTempBucket() throws IOException {
           request.setReadTimeout(60000); // 1 minute read timeout
         };
     Storage storage =
-        new Storage.Builder(new NetHttpTransport(), new JacksonFactory(), requestInitializer)
+        new Storage.Builder(new NetHttpTransport(), new GsonFactory(), requestInitializer)
```

Comment:
Looks like this is now failing code formatting checks, could you please run: `./gradlew :sdks:java:io:google-cloud-platform:spotlessApply` from the root of the project to fix?

Reply:
I ran this on the cmd line but the build failed

Reply:
![Screenshot from 2024-08-15 21-24-41](https://github.com/user-attachments/assets/287cfc84-4cf7-462a-a4ec-375928514c75)

May be this is happening because of I am missing some project requirements

Reply:
Have you tried going through https://github.com/apache/beam/blob/master/CONTRIBUTING.md#setup-your-environment-and-learn-about-language-specific-setup? In particular, the `./local-env-setup.sh` piece might help

If you don't want to deal with the dependencies for this, you can also just use the suggestion from the build itself - https://github.com/apache/beam/actions/runs/10406736496/job/28820457976?pr=32158

In this case, the relevant piece is:

```
Execution failed for task ':sdks:java:io:google-cloud-platform:spotlessJavaCheck'.
> The following files had format violations:
      sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/healthcare/HttpHealthcareApiClient.java
          @@ -734,8 +734,7 @@
           ????????????????CloudHealthcareScopes.CLOUD_PLATFORM,?StorageScopes.CLOUD_PLATFORM_READ_ONLY));
           
           ????client?=
          -????????new?CloudHealthcare.Builder(
          -????????????????new?NetHttpTransport(),?new?GsonFactory(),?requestInitializer)
          +????????new?CloudHealthcare.Builder(new?NetHttpTransport(),?new?GsonFactory(),?requestInitializer)
           ????????????.setApplicationName("apache-beam-hl7v2-io")
           ????????????.build();
           ????httpClient?=
      sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/healthcare/FhirIOTestUtil.java
          @@ -136,8 +136,7 @@
           ??????????request.setReadTimeout(60000);?//?1?minute?read?timeout
           ????????};
           ????Storage?storage?=
          -????????new?Storage.Builder(new?NetHttpTransport(),?new?GsonFactory(),?requestInitializer)
          -????????????.build();
          +????????new?Storage.Builder(new?NetHttpTransport(),?new?GsonFactory(),?requestInitializer).build();
           ????List<StorageObject>?blobs?=?storage.objects().list(DEFAULT_TEMP_BUCKET).execute().getItems();
           ????if?(blobs?!=?null)?{
           ??????for?(StorageObject?blob?:?blobs)?{
  Run './gradlew :sdks:java:io:google-cloud-platform:spotlessApply' to fix these violations.
```

So to fix, you'd update the lines in `FhirIOTestUtil.java` and `HttpHealthcareApiClient.java` to follow the specified format

Reply:
Hi @damccorm ,
I went through the setup your environment again and again. On my system(ubuntu 22.04) the ./local-env-setup.sh is not working so did all the things manually. Only goavro is something which is not installing though I have search and read a lot on internet but could not find a feasible solution for the same.

Reply:
Ok, could you please just manually edit the lines then? If you share the error you're getting I may be able to help as well

Reply:
The error I am getting is mentioned in the comment above.

Reply:
![Screenshot from 2024-08-17 10-44-05](https://github.com/user-attachments/assets/c8eb879a-1027-41c0-80c9-cd077a4c977a)

This is the error I am getting while running ./local-env-setup.sh and in the comments above I have shown the error I am getting while building the project.

Reply:
Looks like a bunch of permission errors - could you try running `sudo ./local-env-setup.sh` and running it as root? Or if you don't want to give elevated permissions to the whole script, you could modify it to just run certain commands as root.